src/utilities/project-utils.ts

```diff
@@ -10,109 +10,257 @@ import pcLogo from "../assets/project-images/pc-logo.png";
 import geniLogo from "../assets/project-images/geni-logo.png";
 import cbioLogo from "../assets/project-images/connectedbio-logo.png";
 import watersLogo from "../assets/project-images/waters-logo.png";
+import codapLogo from "../assets/project-images/CODAP.png";
+import concordLogo from "../assets/concord.png";
+//partner logos
+import natgeoLogo from "../assets/partner-logos/natgeo-logo.png";
+import ucscLogo from "../assets/partner-logos/ucsc-logo.png";
+import nsfLogo from "../assets/partner-logos/nsf-logo.png";
+import stemCreateLogo from "../assets/partner-logos/stem-create.png";
+import ipumsLogo from "../assets/partner-logos/IPUMS-generic.png";
+import ipumsUsaLogo from "../assets/partner-logos/IPUMS-logo-usa.png";
+import msuWoodmarkLogo from "../assets/partner-logos/MSU-Wordmark-Green-CMYK.png";
+import pennStateLogo from "../assets/partner-logos/PennStateLogo.png";
+import argonneLogo from "../assets/partner-logos/argonne.png";
+import millersvilleLogo from "../assets/partner-logos/millersville.png";
+import uicLogo from "../assets/partner-logos/uic.png";
 
 export interface ProjectType {
   id: number;
   name: string;
-  logo: any;
+  projectLogo: any;
   url: string;
+  footer: string;
+  footerLogo: Array<any>;
+  defaultTheme: string;
```

Comment:
Is this theme parameter needed now?  I think moving forward themes will just be a set of colors and the activity player won't use the old LARA themes that are set in authoring.